bin/gwdetchar-mct

```diff
@@ -25,9 +25,6 @@ import numpy
 import os
 import sys
 
-from matplotlib import (use, rcParams)
-use('agg')
-
```

Comment:
Is this removal just because `gwdetchar-mct` doesn't actually plot anything?

Reply:
@duncanmmacleod, yes, that's right, I figured it was best to remove the redundancy.